packages/salesforcedx-vscode-lwc/test/jest/testSupport/testRunner/testRunner.test.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { getTestNamePatternArgs } from '../../../../src/testSupport/testRunner/testRunner';
+import { TestCaseInfo, TestFileInfo, TestInfoKind, TestType } from '../../../../src/testSupport/types';
+import { Uri } from 'vscode';
+import * as path from 'path';
+
+describe('testRunner Unit Tests.', () => {
+  const mockUriPath = path.join('/');
+  const mockURI = {
+    fsPath: mockUriPath
+  } as Uri;
+
+  describe('test getTestNamePatternArgs', () => {
+    it('Should return testNamePattern if flag is included', async () => {
+      const testName = 'Testing is Fun!';
+      const testExecutionInfo: TestCaseInfo = {
+        kind: TestInfoKind.TEST_CASE,
+        testType: TestType.LWC,
+        testUri: mockURI,
+        testName
+      };
+      const testPatternArgs = getTestNamePatternArgs(testExecutionInfo);
+      expect(testPatternArgs.length).toEqual(2);
+      expect(testPatternArgs).toContain('--testNamePattern');
+      expect(testPatternArgs).toContain(testName);
```

Comment:
These tests seem like a perfect opportunity to use snapshots for the expects.  It will capture the expected output you can verify in the snaps, and if/when we update the regex to handle the weird cases it will be simpler to update the tests.